src/AWS.Deploy.Orchestration/Orchestrator.cs

```diff
@@ -209,14 +209,16 @@ public async Task<bool> CreateContainerDeploymentBundle(CloudApplication cloudAp
                 }
 
                 await _deploymentBundleHandler.BuildDockerImage(cloudApplication, recommendation, imageTag);
+
+                _interactiveService.LogSectionStart("Pushing container image to Elastic Container Registry (ECR)", "Using the docker CLI log on to ECR and push the local image to ECR.");
```

Comment:
Same nit as on `DeployCommand` on adding either "to" or a comma after "CLI"

Reply:
Added `to`